tests/tests_all_purge.yml

```diff
@@ -9,12 +9,14 @@
       login -a -s staff_u sar-user
 
   tasks:
-    - name: Install SELinux tool semanage on Fedora
+    - name: Ensure SELinux tool semanage
```

Comment:
If set_selinux_variables.yml "[ensure SELinux tool semanage](https://github.com/linux-system-roles/selinux/pull/77/files#diff-94524c7699c4ef2c05e1504ca723ff3d38cb0416ff381e7f874a7bc9e6417040R2-R9)" and it's imported at line 37, this `package` is not needed here?

Please note that `package` module is slow even if the package is not downloaded...

Reply:
The `package` is needed here because of https://github.com/linux-system-roles/selinux/blob/f35251fff0873e536d90e108e5b599703ae2138b/tests/tests_all_purge.yml#L29

> Please note that package module is slow even if the package is not downloaded...

ok - but I would like to get this fix in so we can drop the patch we're carrying downstream